src/router/index.ts

```diff
@@ -38,9 +40,37 @@ const router = createRouter({
           path: "user-checkin",
           component: UserCheckIn,
           beforeEnter: authGuard
+        },
+        {
+          path: "/event-register",
+          component: EventRegister
         }
       ]
     },
+    {
+      path: "/test",

```

Comment:
Is something wrong here? Please put every component that needs navbar and footer inside pageSkeleton route.

Reply:
![image](https://user-images.githubusercontent.com/12471844/114375542-f6abc880-9bae-11eb-88d3-e9531ba5a3e4.png)